examples/random_numbers/lib_rand_test.c

```diff
@@ -0,0 +1,61 @@
+/******************************************************************************
+* Psuedo Random Number Generator using a Linear Feedback Shift Register 
+* - Demo Program
+* See the GitHub for more information:
+* https://github.com/ADBeta/CH32V003_lib_rand
+*
+* Ver 2.0    09 Sep 2024
+* 
+* Released under the MIT Licence
+* Copyright ADBeta (c) 2024
+*
+* Permission is hereby granted, free of charge, to any person obtaining a copy
+* of this software and associated documentation files (the "Software"), to
+* deal in the Software without restriction, including without limitation the 
+* rights to use, copy, modify, merge, publish, distribute, sublicense, and/or 
+* sell copies of the Software, and to permit persons to whom the Software is 
+* furnished to do so, subject to the following conditions:
+* The above copyright notice and this permission notice shall be included in 
+* all copies or substantial portions of the Software.
+*
+* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, 
+* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF 
+* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+* DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR 
+* OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE 
+* USE OR OTHER DEALINGS IN THE SOFTWARE.
+******************************************************************************/
+#include "ch32v003fun.h"
+#include "lib_rand.h"
+
+#include <stdio.h>
+
+// NOTE: RANDOM_STRENGTH is defined as 2 in funconfig.h. Change this if needed
+int main() 
+{
+	SystemInit();
+
+	// Optional: Seed the Random Number Generator.
+	// It is pre-seeded with 0x747AA32F
+	seed(0x12345678);
+
+	// Print 4 Random Numbers
+	// These will ALWAYS be:
+	// * 3443170572
```

Comment:
Are these numbers correct? Should they actually be 40-bit (10 hex digit) numbers? Not 32-bit (4 hex digit) numbers?

Reply:
Those are correct decimal values, not hex. 
Assumed representing them in hex was confusing

Reply:
Ah thanks - didn't realise that - that they were decimal. Wouldn't consistent use of Hex be clearer?

Reply:
I've commented it to be a bit more clear now.
I imagine the purpose of pseudo-random numbers in most cases to have decimal numbers as outputs, like bingo/lottery numbers so it seemed more in-keeping with end usecase. 

I'm open to change it if that seems weird